dockyard/cmd/api.py

```diff
@@ -0,0 +1,15 @@
+from wsgiref import simple_server
```

Comment:
It is ok to use this server but we should atleast know, why this server is being used not any other server. It is better, we do a bit of research on it. It will help us in understanding about servers.